lib/src/elastic_client_impl.dart

```diff
@@ -169,14 +169,14 @@ class Aggregation {
 }
 
 class Bucket {
-  dynamic key;
+  Object? key;
```

Comment:
Is there any benefit of using `Object?` vs. `dynamic`?

Reply:
if it was dynamic this line must be changed: `if (key != null) 'key': key!` or the `Map<String, Object>` signature.  I am ok in using dynamic instead of Object.